fgcm/fgcmNumbaUtilities.py

```diff
@@ -1,6 +1,6 @@
 try:
     from numba import jit
-    has_numba = True
+    has_numba = False
```

Comment:
Why try the import at all now?

Reply:
In case I bring this code back in the future, I guess.

Reply:
This doesn't really facilitate that (and I'd argue it's more confusing to leave it like this), but up to you!